sdk/purview/purview-catalog-rest/src/clientDefinitions.ts

```diff
@@ -285,40 +285,40 @@ export interface EntityCreateOrUpdate {
    */
   post(
     options: EntityCreateOrUpdateParameters
-  ): Promise<EntityCreateOrUpdate200Response | EntityCreateOrUpdatedefaultResponse>;
+  ): StreamableMethod<EntityCreateOrUpdate200Response | EntityCreateOrUpdatedefaultResponse>;
```

Comment:
Added this file to show what changes will be needed in the generator. As you can see the changes in the generated code are minimal.